JavaScript/CodingConventions.md

```diff
@@ -180,32 +180,32 @@ When using `switch` statements:
  - Use `break`, `return` or `throw` for each case other than `default`
  - The `default` case should always be last.
 
-As with all our whitespace guidelines, switch statements follow JSLint.
+Use 1 tab indentation for `case` / `default` statements.
 
 ```javascript
 // bad
 switch (foo) {
-    default:
-        z();
-    case 'bar':
-    case 'foobar':
-        x();
-        break;
-    case 'baz':
-        y();
-}
-
-// good
-switch (foo) {
 case 'bar':
 case 'foobar':
-	x();
-	break;
+    x();
```

Comment:
We use tabs in all JS files and it's already described in https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#white-space-guidelines.

Using spaces instead would be a topic for whole another conversation ;)